index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <!-- fontawesme link -->
+    <script src="https://kit.fontawesome.com/fc7e7ce012.js" crossorigin="anonymous"></script>
+    <!-- style css -->
+    <link rel="stylesheet" href="style.css">
+
+
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="navbar">
+                <ul class="top-navbar-menu">
+                    <li><a href="">Facebook</a></li>
+                    <li><a href="">Twitter</a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Log out</a></li>
+                </ul>
```

Comment:
where did the previous reviewer said. pleasssss show me

Reply:
@Esmatullah12 See the comment [here ](https://github.com/Esmatullah12/html-css-capstone-project/pull/1#discussion_r1174303289). I mentioned this. 

I know sometimes, it might be a little uncomfortable to get review comments that insist on some issues being fixed, ultimately it is for your good and better training to be more detailed as a developer.